### ISSUE TITLE
Proposal: increase default arity (or ignore language callbacks)

### ISSUE THREAD

**Issue by christhekeele** [rrrene/credo#475]
### Environment

* Credo version (`mix credo -v`): 0.8.8
* Erlang/Elixir version (`elixir -v`): 1.5.2
* Operating system: OSX

### What were you trying to do?

Add credo to a meta-programming-heavy project

### Expected outcome

`Credo.Check.Refactor.FunctionArity` should not complain about language-enforced arity lengths.

### Actual outcome

`Credo.Check.Refactor.FunctionArity` will always warn on `__on_definition__/6` callbacks.

We could increase the default from 5 to 6, add code to handle these special cases, or provide a way to configure no-oping functions of certain names—either way I don't think this is a valid refactoring suggestion no matter what, so credo should exhibit different behaviour.

**Comment by rrrene** [comment]
Thanks for reporting this 😀  It is now fixed on `master`.

You can try this by setting the Credo dep to

```elixir
{:credo, github: "rrrene/credo"}
```

Please report back if your issue is solved! 👍

**Comment by christhekeele** [comment]
:+1: that does it, thanks!
